.github/workflows/android_deploy_production_to_playstore.yml

```diff
@@ -0,0 +1,61 @@
+name: Android - Deploy Production build to Play Store
+
+on:
+  push:
+    branches:
+      - main
+      - feature/61-setup-cd-to-build-and-deploy-android-app-to-playstore
```

Comment:
@luongvo shall we remove this temporary branch since the CI passed? 😄

Reply:
Of course, I missed that 😛

Reply:
Fixed in https://github.com/nimblehq/flutter_templates/pull/163/commits/a9fa35904a66098fcf5d5dce0b21d15227d47860